utils/serde/src/main/java/io/apicurio/registry/utils/serde/AbstractKafkaSerDe.java

```diff
@@ -48,12 +51,19 @@
     public static final String REGISTRY_ID_HANDLER_CONFIG_PARAM = "apicurio.registry.id-handler";
     public static final String REGISTRY_CONFLUENT_ID_HANDLER_CONFIG_PARAM = "apicurio.registry.as-confluent";
 
+    // Constants for using headers to store the ids
+    public static final String USE_HEADERS = "apicurio.use.headers";
```

Comment:
I think this should be `apicurio.registry.use.headers` to match the other config prop names.

Reply:
Agree